arc/portal_gallery_package/Manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+   "$schema": "https://gallery.azure.com/schemas/2015-04-01/manifest.json#",
+   "name": "AzureKeyVaultSecretsProvider",
+   "publisher": "Microsoft",
+   "version": "1.0.0-preview",
+   "displayName": "Azure Key Vault Secrets Provider (preview)",
+   "publisherDisplayName": "Microsoft",
+   "publisherLegalName": "Microsoft",
+   "summary": "Azure Key Vault Secrets Provider provides integration of Azure Key Vault with a Kubernetes cluster",
```

Comment:
The Azure Key Vault Provider for Secrets Store CSI Driver allows for the integration of Azure Key Vault as a secrets store with a Kubernetes cluster via a [CSI volume](https://kubernetes-csi.github.io/docs/).

ref: https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-akv-secrets-provider and https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver

Reply:
Yes. I am referring our docs. But this change is to satisfy regex requirements while package this. It only allows 100 chars for summary.